keps/sig-node/4205-psi-metric/README.md

```diff
@@ -284,8 +284,10 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
     - PSI metrics collection doesn't introduce excessive CPU or memory usage increase in the kubelet
 
 #### GA
-- Gather evidence of real world usage.
-- No major issue reported.
+- Quantify the cAdvisor and kubelet-level overhead of PSI metric collection, especially where PSI is disabled at the kernel level.
```

Comment:
Let's also ensure we cover stress testing scenario. We may need to expand on top of the [previous performance benchmarking](https://docs.google.com/document/d/1hBc-mNpC8hNMoN70jsRtbr5sdLM6F2foWWtWdJiJzic/edit?usp=sharing)